Git Formats/Git Code Owners.sublime-syntax

```diff
@@ -46,6 +50,18 @@ contexts:
         - Git Common.sublime-syntax#email-meta
         - Git Common.sublime-syntax#email-name
 
+  comments:
+    - match: \s*(?={{comment_char}})
+      set: comment-body
+
+  comment-body:
+    - meta_include_prototype: false
+    - match: ({{comment_char}}).*\n?
+      scope: comment.line.git.codeowners
+      captures:
+        1: punctuation.definition.comment.git.codeowners
+      pop: 1
+
```

Comment:
```suggestion
  comments:
    - match: \s*(?={{comment_char}})
      set: comment-start

  comment-start:
    - meta_include_prototype: false
    - match: {{comment_char}}
      scope: punctuation.definition.comment.git.codeowners
      push: comment-body

    comment-body
      - meta_scope: comment.line.git.codeowners
      - match: \n?$
        pop: 1
```

Reply:
How about this?

Reply:
- Shouldn't the `push` be a `set`? `comment-start` will remain on top of the stack otherwise.
- Probably doesn't matter, but why `\n$?` instead of `$\n?`?

Reply:
Solution may be way simpler, by just tweaking word boundary patterns

see: https://github.com/mataha/Packages/pull/1

Reply:
Would changing `#;` to `#` only in CODEOWNERS make sense?

Reply:
This would mean to duplicate comment and email implementation. IMHO chances for an email or name to contain ; in its name is low enough to avoid those extra efforts.